src/main/java/org/mitre/synthea/export/CSVExporter.java

```diff
@@ -150,58 +160,80 @@ private CSVExporter() {
         outputDirectory = outputDirectory.resolve(subfolderName);
         outputDirectory.toFile().mkdirs();
       }
+      
+      String includedFilesStr = Config.get("exporter.csv.included_files", "").trim();
+      String excludedFilesStr = Config.get("exporter.csv.excluded_files", "").trim();
+      
+      List<String> includedFiles = new ArrayList<>();
+      List<String> excludedFiles = new ArrayList<>();
+      
+      if (!includedFilesStr.isEmpty() && !excludedFilesStr.isEmpty()) {
+        System.err.println(
+            "CSV exporter: Included and Excluded file settings are both set -- ignoring both");
+      } else {
+        if (!includedFilesStr.isEmpty()) {
+          List<String> files = Arrays.asList(includedFilesStr.split(","));
+          // normalize filenames -- trim, lowercase, add .csv if not included
+          files = files.stream().map(f -> f.trim().toLowerCase())
+              .map(f -> f.endsWith(".csv") ? f : f + ".csv").collect(Collectors.toList());
+          includedFiles.addAll(files);
+
+          if (!includedFiles.contains("patients.csv")) {
+            System.err.println("WARNING! CSV exporter is set to not include patients.csv!");
+            System.err.println("This is probably not what you want!");
+          }
+
+        } else {
+          List<String> files = Arrays.asList(excludedFilesStr.split(","));
+          // normalize filenames -- trim, lowercase, add .csv if not included
+          files = files.stream().map(f -> f.trim().toLowerCase())
+              .map(f -> f.endsWith(".csv") ? f : f + ".csv").collect(Collectors.toList());
+          excludedFiles.addAll(files);
```

Comment:
This is essentially the same a lines 175 - 179. You could just create a lambda to encapsulate the repeated functionality right in the method.

Reply:
I tried it out and lambdas in Java are just ugly enough that I went for a full helper method instead